code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -304,19 +304,19 @@
 		if(op != "")
 			switch(op)
 				if("+")
-					result += val
+					result = (result + val)
```

Comment:
Please do comment, our reviewers don't pay enough attention to these small details. In a year someone will fix this and it will get merged.